### PR TITLE
Notice text and license mapping for chunk2

### PR DESCRIPTION
**Pull request by robbavey**

### CODE REVIEW

File: tools/dependencies-report/src/main/resources/licenseMapping.csv [hunk 1]
```diff
@@ -42,3 +42,40 @@ dependency,dependencyUrl,licenseOverride
...
+"hitimes:1.2.6",https://github.com/copiousfreetime/hitimes,ISC
+"http-cookie:1.0.3",https://github.com/sparklemotion/http-cookie,MIT
+"http-form_data:1.0.1",https://github.com/httprb/form_data,MIT
+"http:0.9.9",https://github.com/httprb/http,MIT
+"http_parser.rb:0.6.0",https://github.com/tmm1/http_parser.rb,MIT
+"i18n:0.6.9",https://github.com/svenfuchs/i18n,MIT
+"insist:1.0.0",https://github.com/jordansissel/ruby-insist,Apache-2.0
+"com.headius:invokebinder:1.7",https://github.com/headius/invokebinder,Apache-2.0
+"jar-dependencies:0.3.12",https://github.com/mkristian/jar-dependencies,MIT
+"org.jruby:jay-yydebug:-",https://github.com/jruby/jay-yydebug,EPL-2.0|GPL-2.0|LGPL-2.1
```
Comment (robbavey) [inline tools/dependencies-report/src/main/resources/licenseMapping.csv]: This library is referenced in [jruby licensing text](https://github.com/jruby/jruby/blob/9.1.13.0/COPYING), but I could not find the version used anywhere, not even inside the `jruby-complete.jar`

File: tools/dependencies-report/src/main/resources/licenseMapping.csv [hunk 2]
```diff
@@ -42,3 +42,40 @@ dependency,dependencyUrl,licenseOverride
...
+"org.jruby.jcodings:jcodings:1.0.18",https://github.com/jruby/jcodings,MIT
+"jdbc-derby:10.12.1.1",https://github.com/jruby/activerecord-jdbc-adapter,Apache-2.0
+"com.github.jnr:jffi:1.2.16",https://github.com/jnr/jffi,Apache-2.0|LGPL-3.0
```
Comment (robbavey) [inline tools/dependencies-report/src/main/resources/licenseMapping.csv]: For dual/triple licenses, not clear whether the GPL/LGPL licenses come under `xxx-only` or `xxx-or-later` licenses

File: tools/dependencies-report/src/main/resources/notices/ftw-0.0.48-NOTICE.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+Apache License (2.0)
```
Comment (robbavey) [inline tools/dependencies-report/src/main/resources/notices/ftw-0.0.48-NOTICE.txt]: https://github.com/jordansissel/ruby-ftw/blob/7d9ea51c7e92ecd4f4cd1b4cb0bd2c43ea497f23/ftw.gemspec#L19

File: tools/dependencies-report/src/main/resources/notices/filewatch-0.9.0-NOTICE.txt [hunk 4]
```diff
@@ -0,0 +1 @@
+License is MIT (very permissive).
```
Comment (robbavey) [inline tools/dependencies-report/src/main/resources/notices/filewatch-0.9.0-NOTICE.txt]: https://github.com/jordansissel/ruby-filewatch/blob/master/LICENSE

File: tools/dependencies-report/src/main/resources/notices/jay-yydebug-NOTICE.txt [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  GNU Lesser General Public License version 2.1
+
+bytelist (http://github.com/jruby/bytelist),
+jnr-posix (https://github.com/jnr/jnr-posix),
+jruby-openssl (https://github.com/jruby/jruby-openssl),
+jruby-readline (https://github.com/jruby/jruby-readline),
+psych (https://github.com/ruby/psych),
+yydebug (https://github.com/jruby/jay-yydebug/)
+are released under the same copyright/license.
+
```
Comment (robbavey) [inline tools/dependencies-report/src/main/resources/notices/jay-yydebug-NOTICE.txt]: This section extracted from https://github.com/jruby/jruby/blob/master/COPYING

File: tools/dependencies-report/src/main/resources/notices/jnr-ffi-2.1.7-NOTICE.txt [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This code is distributed in the hope that it will be useful, but WITHOUT
+  ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+  FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+  version 3 for more details.
+
+  You should have received a copy of the GNU Lesser General Public License
+  version 3 along with this work.  If not, see <http://www.gnu.org/licenses/>.
```
Comment (robbavey) [inline tools/dependencies-report/src/main/resources/notices/jnr-ffi-2.1.7-NOTICE.txt]: Notice does not include the GPL v3 referenced here

File: tools/dependencies-report/src/main/resources/notices/gradle-license-report-0.7.1-NOTICE.txt [hunk 7]
```diff
@@ -0,0 +1,201 @@
+                               Apache License
```
Comment (yaauie) [inline tools/dependencies-report/src/main/resources/notices/gradle-license-report-0.7.1-NOTICE.txt]: I find it interesting that a project whose sole purpose is to report on licenses fails to correctly apply the license to itself by following the instructions in the Apache License's "How to apply the Apache License to your work".

**Comment by jsvd** [comment]
merged in https://github.com/elastic/logstash/pull/9886

**Pull request closed without merging**